labels.go

```diff
@@ -152,14 +152,32 @@ func PrevLabel(s string, n int) (i int, start bool) {
 	if n == 0 {
 		return len(s), false
 	}
-	lab := Split(s)
-	if lab == nil {
-		return 0, true
+
+	l := len(s) - 1
+	if s[l] == '.' {
```

Comment:
This needs a check that `l >= 0` otherwise it will panic for an empty string.

Reply:
fixed